.github/workflows/clang_sanitizers.yml

```
@@ -147,14 +149,14 @@ jobs:
 
         export OMP_NUM_THREADS=2
```

```suggestion
        export OMP_TOOL_LIBRARIES=/usr/lib/llvm-17/lib/libarcher.so
        export OMP_NUM_THREADS=2
```
Could you please try this potential fix ?